src/Razor/src/Microsoft.AspNetCore.Razor.ProjectEngineHost/Serialization/RazorSerializationConstants.cs

```diff
@@ -1,9 +1,7 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the MIT license. See License.txt in the project root for license information.
 
-#nullable disable
```

Comment:
I didn't aim to nullable enable this new project, as the PR is big enough already. Will log a follow up to fix it properly.